<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DTM</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="Draw" name="Canvas" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Generate points</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="points_count">
       <property name="text">
        <string>199</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Randomised Grid</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Kupa</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Spočinek</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Údolí</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hřbet</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Points">
       <property name="text">
        <string>Points</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_8">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Delaunay">
       <property name="text">
        <string>Delaunay</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Interval of contours</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="DZ">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Contours">
       <property name="text">
        <string>Contours</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_7">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_2">
       <item>
        <property name="text">
         <string>Slope</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Aspect</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Analyze">
       <property name="text">
        <string>Analyze</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Clear">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Draw</class>
   <extends>QWidget</extends>
   <header location="global">draw.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
